spec/factories/factory_bot.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'factory_bot'
+require 'faker'
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+end
+
+FactoryBot.define do
+  factory :book do
+    genre { Faker::Book.genre }
+    author { Faker::Book.author }
+    image { Faker::File.file_name('foo/bar', 'baz', 'jpg') }
+    editorial { Faker::Book.publisher }
+    year { Faker::Date.backward }
```

Comment:
year field its string, but backward generate a Date, no? Review it please 😉

Reply:
shoud i  keep the string type ? date type sounds better

Reply:
i took the string option using .year.to_s methods